certification/pyxis/layers.go

```diff
@@ -0,0 +1,71 @@
+package pyxis
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	cranev1 "github.com/google/go-containerregistry/pkg/v1"
+
```

Comment:
Nit: can we organize the imports?

Reply:
ctrl + option + O didn't do anything 🤷  but I removed the spaces..

Reply:
Run 'make fmt'